examples/with-logger/renderer/pages/home.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
```

Comment:
In my opinion define a const type NextPage is more readable and compliant with typescript BP.
```suggestion
import { NextPage } from "next";
import React from "react";
```